### PR TITLE
fix: `no_superfluous_phpdoc_tags` must honor multiline docs

### LINKED ISSUE #7692: `no_superfluous_phpdoc_tags` removes actual docs when these are after phpDoc tag
## Bug report

My guess is, php-cs-fixer detects the `@return void` as useless, but then decides to delete the whole comment instead of just the single tag from the phpdoc

### Description

a phpdoc is removed, which should be preserved

### Runtime version

```
$ php php-cs-fixer -V
PHP CS Fixer 3.46.1-DEV Three Keys by Fabien Potencier, Dariusz Ruminski and contributors.
PHP runtime: 8.2.12
```

using https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/commit/3566e7c926ada331cd07cf6436947d0d89a1d216

### Used command

`php php-cs-fixer fix --dry-run test.php --diff -vvv`

### Configuration file

default config from PHP-CS-Fixer repo

### Code snippet that reproduces the problem

```php
<?php

class X {
    /**
     * @return void
     *
     * Validate price information values
     */
    private function validatePriceinformationConfigset(): void
    {

    }
}
```

expected result

```php
<?php

class X {
    /**
     * Validate price information values
     */
    private function validatePriceinformationConfigset(): void
    {

    }
}
```

actual result

```php
<?php

class X {
    private function validatePriceinformationConfigset(): void
    {

    }
}
```

### PR DESCRIPTION
**Pull request by mvorisek**
fix #7692

### CODE REVIEW

**Comment by mvorisek** [comment]
@staabm can you please confirm this fix fixes all your issues?

Also thank you for https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/7692#issuecomment-1881080467 test case, it was much more easier to look into it than into https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/7692#issuecomment-1880695655 with many different fixers applied in consequence.

File: tests/Fixer/Phpdoc/NoSuperfluousPhpdocTagsFixerTest.php [hunk 1]
```diff
@@ -2528,6 +2528,30 @@ static function ($foo): int { return 1; };',
 /** @return int */
 static fn ($foo): int => 1;',
         ];
+
+        yield 'multiline @param must be kept even if there is no description on the phpdoc tag line' => [
+            <<<'EOD'
```
Comment (julienfalque) [inline tests/Fixer/Phpdoc/NoSuperfluousPhpdocTagsFixerTest.php:2533]: Not a big deal but other test cases here don't use NOWDOC syntax. I would not use it for new cases for consistency.

(Maybe we should create a new fixer for that.)

**Review by julienfalque — COMMENTED** [review]
:+1: 

Please add a test case with a `@var` tag on a property.

File: tests/Fixer/Phpdoc/NoSuperfluousPhpdocTagsFixerTest.php (hunk 1 above)
Comment (Wirone) [reply tests/Fixer/Phpdoc/NoSuperfluousPhpdocTagsFixerTest.php:2533]: @julienfalque #7665 😉

Comment (mvorisek) [reply tests/Fixer/Phpdoc/NoSuperfluousPhpdocTagsFixerTest.php:2533]: yes :)

**Comment by mvorisek** [comment]
> 👍
> 
> Please add a test case with a `@var` tag on a property.

done

**Review by julienfalque — APPROVED** [review]

**Review by Wirone — APPROVED** [review]

**Comment by staabm** [comment]
hey every,

thanks for the fast fix. I just checked my 2 local reproducer from #7692 and can confirm these are no longer a problem.

thanks again!

**Pull request merged**